persistence/src/test/java/cz/muni/fi/pa165/mushrooms/dao/VisitDaoTest.java

```diff
@@ -151,9 +148,237 @@ public void create_validVisit() throws Exception {
         em.persist(testVisit);
 
         assertThat(visitDao.findAll().size()).isEqualTo(sizeBeforeCreate + 1);
```

Comment:
Shouldn't be "FindAll() test" before this one, when you use the function in other tests? Respectively, shouldn't it be the first of all tests?

Reply:
ok, moved to the top of the "find" methods